src/cli/cli_syntax.md

```diff
@@ -1,22 +1,143 @@
-## CLI Syntax
+# CLI Syntax
+Before you start reading this chapter I highly recommend you the read the [POSIX Utility Conventions][POSIX Utility Conventions]
+it is about 10-20 min read which will provide a solid base when it comes to clis and their syntax.
 
-### Elements in a cli (Terminology)
-** TBD **
+## Terminology
 
-### Tokens commonly used in CLI syntax
-** TBD **
+If you read about command lines and command line interfaces usually you will find a specific kind of terminology
+(*command, utility, subcommand, argument, option, flag, switch, operand*) even though the meaning does slightly differ
+depending which source or in which context a term is mentioned.
+To make sure we are all on the same page regarding this terms, please read the following table,
+which describes the terms and their meaning in the context of this book.
 
-### Posix CLI syntax
-** TBD **
 
-### Unix "Common"
-** TBD **
+| Terminology of this Book           | Description                                                       | Names in other standards/definitions |
+|------------------------------------|-------------------------------------------------------------------|--------------------------------------|
+| **command**                        | a cli tool / command                                              | utility (Posix)                      |
+| **subcommand**                     | a subcommand of a command/cmd group                               |                                      |
+| **argument**-(required/positional) | required/positional-argument                                      | operand (Posix)                      |
```

Comment:
Don't need a hyphen for argument here:
"**argument** (required/positional) | required/positional argument"

I get that you're trying to make it all one connected word, to make it obvious it all refers to one thing when you use it later in the book. For that reason you could also decide to ignore this comment and keep them connected via a hyphen. It does then make it obvious that **required-argument** or **positional-argument** refers to just one thing.